app/models/facility.rb

```diff
@@ -168,6 +168,10 @@ def child_region_type
     nil
   end
 
+  def label_with_district
+    "#{name} (#{facility_group.name})"
```

Comment:
Shall we use `district_region` instead? Start relying more on regions and less on FGs?